### PR TITLE
Install TensorRT lib on oss docker and enable fx2trt unit test

### PR DESCRIPTION
**Pull request by wushirong**
Summary: CI

Differential Revision: D33241684

Lib installed and unit test run on https://github.com/pytorch/pytorch/actions/runs/1604076060

### CODE REVIEW

File: .jenkins/pytorch/test.sh [hunk 1]
```diff
@@ -562,6 +568,8 @@ elif [[ "${BUILD_ENVIRONMENT}" == *distributed* || "${JOB_BASE_NAME}" == *distri
   test_rpc
 elif [[ "${TEST_CONFIG}" = docs_test ]]; then
   test_docs_test
+elif [[ "${BUILD_ENVIRONMENT}" == *linux-xenial-cuda11.3-py3.6-gcc7* ]]; then
```
Comment (janeyx99) [inline .jenkins/pytorch/test.sh:571]: Could we base this on TEST_CONFIG being fx2trt instead? Something like 
```suggestion
elif [[ "${TEST_CONFIG}" == fx2trt ]]; then
```

**Review by janeyx99 — APPROVED** [review]
Looks good! Just one comment for better posterity (cuz once we migrate off of 11.3 we want to still run these tests)

**Review by malfet — APPROVED** [review]

**Comment by suo** [comment]
This PR broke macos builds, I am reverting: http://hud2.pytorch.org/commit/dab3d3132bad1070c225ee4a2e8dd5a561cd7d1a

**Comment by wushirong** [comment]
@rerun -l ciflow/trunk

**Comment by suo** [comment]
This PR appears to have added an entirely new job that runs all the unit tests again and takes ~1h30m. Is there a reason we added a new job, as opposed to enabled fx2trt tests on the existing gpu linux jobs?

**Pull request closed without merging**